tests/fast-integration/skr-test/test-own-cluster.js

```diff
@@ -0,0 +1,69 @@
+const {gatherOptions} = require('./index');
+const {getOrProvisionSKR} = require('./provision/provision-skr');
+const {GardenerClient, GardenerConfig} = require('../gardener');
+
+const {getSecret} = require('../utils');
+
+const {expect} = require('chai');
+
+const {toBase64} = require('../utils');
+
+const {genRandom} = require('../utils');
+
+const skipProvisioning = process.env.SKIP_PROVISIONING === 'true';
+const provisioningTimeout = 1000 * 60 * 30; // 30m
+const deprovisioningTimeout = 1000 * 60 * 95; // 95m
+let globalTimeout = 1000 * 60 * 70; // 70m
+const slowTime = 5000;
+
+const gardener = new GardenerClient(GardenerConfig.fromEnv());
+
+describe('SKR own_cluster test', function() {
+  if (!skipProvisioning) {
+    globalTimeout += provisioningTimeout + deprovisioningTimeout;
```

Comment:
does it make sense to have this conditionally? this test appears to always create shoot and delete the shoot

Reply:
Removed.